### PR TITLE
update url that moved, update on how to improve scaling for sklearnex

### PR DESCRIPTION
**Pull request by BethanyL**
## Description

- Updated URL that pointed to an Intel page that moved. 
- Updated a warning we have so that it includes the current fix. Added that fix to the code example.
- Removed unnecessary `module use /soft/modulefiles` from code example.

## Screenshots (if applicable)
N/A

## Related Issue(s)
N/A

## Type of Change

- [x] Documentation content update (new page, formatting/typo changes, adding more info, etc.)
- [ ] Functionality bug fix
- [ ] New feature (`mkdocs` feature, `mkdocs-material` style changes, HTML/CSS/JS customization, developer or repo tool)

## Checklist

- [x] I have run `make serve` or `make build-docs` locally and verified that my changes render correctly
- [x] I have added at least one Label to this PR

### CODE REVIEW

**Comment by BethanyL** [comment]
I converted this PR to a draft. I realized that Intel Extension for scikit-learn is not part of the newest frameworks module, and I'm not sure yet how to properly install it.

**Comment by ethanglaser** [comment]
After discussion, it would be best to change all "Intel Extension for Scikit-learn" to just "Extension for Scikit-learn". Could mention it as being "formerly Intel(R) Extension for Scikit-learn" in first reference.

**Comment by felker** [comment]
@ethanglaser is the trailing asterisk in "Extension for Scikit-learn*" a vestige from when Intel was in the name? or something else?

e.g. https://uxlfoundation.github.io/scikit-learn-intelex/latest/index.html

**Comment by napetrov** [comment]
> @ethanglaser is the trailing asterisk in "Extension for Scikit-learn*" a vestige from when Intel was in the name? or something else?
> 
> 
> 
> e.g. https://uxlfoundation.github.io/scikit-learn-intelex/latest/index.html

Yes. Also as a note - we still have to figure out new name so it would be reflected in package name as well.
